### PR TITLE
test: add signing e2e tests

### PR DESCRIPTION
**Pull request by matthewwalsh0**
**Description**

Create Detox E2E tests to verify signing messages via the test dApp.

**Checklist**

* [ ] There is a related GitHub issue
* [x] Tests are included if applicable
* [x] Any added code is fully documented

### CODE REVIEW

**Review by OGPoyraz — DISMISSED** [review]
LGTM

File: e2e/specs/confirmations/sign-messages.spec.js
Comment (bergarces) [inline e2e/specs/confirmations/sign-messages.spec.js:1]: I'm guessing we are waiting for better tools before we do extra assertions to check that they are really returning the signature and not erroring?

**Review by bergarces — DISMISSED** [review]

File: e2e/specs/confirmations/sign-messages.spec.js
Comment (matthewwalsh0) [reply e2e/specs/confirmations/sign-messages.spec.js:1]: Exactly, as soon as Detox can interact with the browser elements, we can verify the text itself from the output.

File: e2e/specs/confirmations/sign-messages.spec.js [hunk 1]
```diff
@@ -7,8 +7,15 @@ import { TestDApp } from '../../pages/TestDApp';
 import SettingsView from '../../pages/Drawer/Settings/SettingsView';
 import AdvancedSettingsView from '../../pages/Drawer/Settings/AdvancedView';
 import { Smoke } from '../../tags';
+import TestHelpers from '../../helpers';
+
+const MAX_ATTEMPTS = 3;
 
 describe(Smoke('Sign Messages'), () => {
+  beforeAll(async () => {
+    jest.setTimeout(150000);
```
Comment (bergarces) [inline e2e/specs/confirmations/sign-messages.spec.js:16]: Do we need so much time?

**Review by bergarces — APPROVED** [review]

File: e2e/specs/confirmations/sign-messages.spec.js (hunk 1 above)
Comment (matthewwalsh0) [reply e2e/specs/confirmations/sign-messages.spec.js:16]: This was standard on every other smoke test I believe so I just added for consistency.

**Pull request merged**